pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -192,6 +198,38 @@ def create_advisory(self, type: str, kind: str, impetus: str) -> int:
         advisory_num = int(match[1])
         return advisory_num
 
+    def get_advisories(self) -> Dict[str, int]:
+        repo = self.working_dir / "ocp-build-data-push"
+        shutil.rmtree(repo, ignore_errors=True)
+        # shallow clone ocp-build-data
+        cmd = [
+            "git",
+            "-C",
+            self.working_dir,
+            "clone",
+            "-b",
+            self.group_name,
+            "--depth=1",
+            self.config["build_config"]["ocp_build_data_repo_push_url"],
+            "ocp-build-data-push",
+        ]
+        _LOGGER.debug("Running command: %s", cmd)
+        subprocess.run(cmd, check=True, universal_newlines=True, cwd=self.working_dir)
+        # update advisory numbers
+        with open(repo / "group.yml", "r") as f:
+            group_config = f.read()
+        
+        advisories = {}
+        for kind in ["rpm", "image", "extras", "metadata"]:
+            match = re.search(
+                fr"^\s+{kind}:\s*([0-9]+)$", group_config, flags=re.MULTILINE
+            )
```

Comment:
I think it is better to use yaml module to parsee group.yml here so we don't need to hardcode the advisory kinds.

Reply:
Done. save_advisories can also use that but going to defer that for the future.